src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_PROPERTY_VALUE = 100;
+    private static final int NUMBER_OF_FIGURES = 5;
+    private String color = new ColorSupplier().getRandomColor();

```

Comment:
```suggestion
    private String color = new ColorSupplier();

```

Reply:
generate random color in `getRandomFigure()` so we have different one each time we call method